modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/BulkResourceManager.java

```diff
@@ -80,6 +83,19 @@ public SCIMResponse processBulkData(String data, UserManager userManager) {
             bulkRequestProcessor.setFailOnError(bulkRequestDataObject.getFailOnErrors());
             bulkRequestProcessor.setUserManager(userManager);
 
+            int maxOperationCount =
```

Comment:
Shall we have a default value in the code level? If it is not enabled in the config, let's take the default value

Reply:
And shall we add a debug logs to log the maxOpCount?

Reply:
I reviewed the situation of removing the default max operation count config from default.json, and during server startup, these configurations are registered to CharonConfig through here [link](https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/blob/998419e530a049a39d0a61b5e31e2c7c2d319d56/components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/IdentitySCIMManager.java#L192). As a result, if the default value is not set or is not a valid integer, it throws an error. Therefore, I haven't added a default value at the code level since a default value is mandatory in this context.